src/utils/keycloak.example.json

```diff
@@ -0,0 +1,7 @@
+{
+    "initOptions": {
+    "url": "https://sso.pathfinder.gov.bc.ca/auth",
```

Comment:
This has to be different for dev/test/prod - so it can't be in a hard coded json file. Please move to environment variables.

Reply:
@Sybrand this is an example file and it was added only to run the app locally.

Reply:
Can you then change it to `https://sso-dev.pathfinder.gov.bc.ca/auth`

Reply:
ok

Reply:
My bad - sorry @Nkuruba